src/main/java/teammates/logic/api/Logic.java

```diff
@@ -116,6 +119,16 @@ public List<String> updateReadNotifications(String googleId, String notification
         return accountsLogic.updateReadNotifications(googleId, notificationId, endTime);
     }
 
+    public List<FeedbackSessionLogEntryAttributes> getFeedbackSessionLogs(String courseId, String email,
+                                                                          long startTime, long endTime, String fsName) {
```

Comment:
Try not to apply an indentation where the parameters are squished all the way to the right like this. Applies everywhere in this PR.